src/components/movies/reactions-display.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Button, Col, Container, Row } from "react-bootstrap";
+import { useDispatch, useSelector } from "react-redux";
+import { userDataSelector } from "../../store/auth/selectors";
+import { postReaction } from "../../store/movie/actions";
+import { selectedMovieSelector } from "../../store/movie/selectors";
+
+export const ReactionsDisplay = ({ userId, movieId }) => {
+    const dispatch = useDispatch();
+
+    const authenticatedUser = useSelector(userDataSelector);
+    const selectedMovie = useSelector(selectedMovieSelector);
+
+    const handleLike = () => {
+        dispatch(
+            postReaction({
+                userId: authenticatedUser.id,
+                movieId: selectedMovie.id,
+                reaction: "like",
+            })
+        );
+    };
+
+    const handleDislike = () => {
+        dispatch(
+            postReaction({
+                userId: authenticatedUser.id,
+                movieId: selectedMovie.id,
+                reaction: "dislike",
+            })
+        );
+    };
+
+    return (
+        <Container>
+            <Row>
+                <Col></Col>
+                {/* ovde postoji problem ako koristim podatke iz store-a - ucitaju se tek nakon reload-a stranice */}
```

Comment:
Ne znam sto ti se to desava, ali ako koristis state na nacin kako bi trebalo, to ne bi smelo da se desava. Trebalo bi da ti se uvek refreshuje komponenta koja koristi neki deo state-a koji je u tom momentu update-ovan. Mozemo videti oko ovoga detaljnije sta se tacno desava.

Reply:
Sigurno moze da se namesti kako treba da kada god se film selektuje, ti taj film u state-u cuvas kao selektovani i onda iz tog objekta samo vadis sta ti treba od property-ja.

Reply:
Malo sam cackao po kodu, pa sam skontao sta ti uzrokuje ovaj problem. Malo je sneaky bug, fino ti se uvukao.

Evo ti hint na slici sta ti pravi problem, verujem da ces skontati sta se dogadja uz jos nekoliko console.logova 😄 
<img width="620" alt="Screenshot 2022-07-18 at 15 13 32" src="https://user-images.githubusercontent.com/44477868/179519078-add16f9f-94f6-47ae-affd-81bf08370479.png">

Reply:
Ne znam da li sam resio kako si ti kontao, ali je reseno.